dev/make-distribution.sh

```diff
@@ -151,7 +151,7 @@ export MAVEN_OPTS="${MAVEN_OPTS:--Xmx2g -XX:ReservedCodeCacheSize=512m}"
 # Store the command as an array because $MVN variable might have spaces in it.
 # Normal quoting tricks don't work.
 # See: http://mywiki.wooledge.org/BashFAQ/050
-BUILD_COMMAND=("$MVN" -T 1C clean package -DskipTests $@)
+BUILD_COMMAND=("$MVN" -T 1C package -DskipTests $@)
```

Comment:
Whoops, had done this for faster rebuilds and forgot to take it out. Will undo this change in the next revision.